gemrb/GUIScripts/GUIRECCommon.py

```diff
@@ -372,7 +372,7 @@ def PlaySoundPressed():
 			break
 	else:
 		NextSound()
-	GemRB.PlaySound (VoiceSet + SoundSeq[SoundIndex], "CHARACT" + str(pc - 1), 0, 0, 5)
+	GemRB.PlaySound (VoiceSet + SoundSeq[SoundIndex], "CHARACT" + str(pc - 1), 0, 0, 4)
```

Comment:
Ideally we should just add the constants to ie_sounds.py or somesuch and use the named ones.